features/lockfiles/ci/build.py

```diff
@@ -58,11 +58,10 @@ def rm(path):
     _, pkg_ref = build_order[0][0]
     pkg_ref = pkg_ref.split("#", 1)[0]
     print("\n********** Rebuild affected package: %s ***********" % pkg_ref)
-    run("conan graph clean-modified release/")
```

Comment:
This change is not for this PR.

Reply:
I know, but Conan 1.18.0 is ready and without this change will break this PR. OR, we could create a new PR only with this change.

Reply:
fixed by https://github.com/conan-io/examples/pull/29